packages/google_sign_in/example/lib/main.dart

```diff
@@ -79,12 +81,12 @@ class SignInDemoState extends State<SignInDemo> {
   String _pickFirstNamedContact(Map<String, dynamic> data) {
     final List<Map<String, dynamic>> connections = data['connections'];
     final Map<String, dynamic> contact = connections?.firstWhere(
-      (Map<String, dynamic> contact) => contact['names'] != null,
+          (Map<String, dynamic> contact) => contact['names'] != null,
```

Comment:
hm, I was assuming this was standard config with the plugin, I'll have to make sure. Thanks.